Gemfile

```diff
@@ -118,3 +118,5 @@ group :test do
   # axe-core for running automated accessibility checks
   gem "axe-core-rspec"
 end
+
+gem "aws-sdk-cloudwatch", "~> 1.81"
```

Comment:
Could we move this up to where our other prod gems are listed?

Reply:
Ooh good spot...

Reply:
Done!